Scripts/generateDocumentation.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+TARGET=$1
+
+# Move to project root directory
+scriptDir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd "$scriptDir/../"
+
+
+if [[ "$TARGET" = "StreamChat" ]];then
+    TARGET_DIRECTORY="Sources/StreamChat"
+elif [[ "$TARGET" = "StreamChatUI" ]]; then
+    TARGET_DIRECTORY="Sources/StreamChatUI"
+else
+    echo "Please specify target to generate docs for (StreamChat or StreamChatUI)"
+    exit 1
+fi
+
+IGNORED_PATHS='Resources\|__Snapshots__\|Tests\|Generated'
+
+# As of May 2021, swift-doc doesn't support directories, so we find all the subdirectories instead
+find "$TARGET_DIRECTORY" -type d -print | grep -v -e $IGNORED_PATHS > /tmp/StreamChat_folder_directories.txt
+
+# Let's run the documentation inside all subdirectories in the target and directory so we can mirror the directories structure
+while read directory; do
+  swift doc generate $directory -n StreamChatUI  -o Documentation/$directory 
```

Comment:
`-n StreamChatUI` 😱 oh no, anyway this will be fixed in next PR for making compatible with docusaurus: https://stream-io.atlassian.net/browse/CIS-875?atlOrigin=eyJpIjoiNTM3MTQ3ZThiNjhiNDc4YWEwOWFjMTZmNDE2YmQxZWEiLCJwIjoiaiJ9 You can run this code inside branch with `git checkout experiment && cd stream-chat-swift-docs && yarn run start`